src/internal/packager/images/pull.go

```diff
@@ -354,24 +356,37 @@ func CleanupInProgressLayers(ctx context.Context, img v1.Image) error {
 
 // SaveSequential saves images sequentially.
 func SaveSequential(ctx context.Context, cl clayout.Path, m map[transform.Image]v1.Image) (map[transform.Image]v1.Image, error) {
+	l := logger.From(ctx)
 	saved := map[transform.Image]v1.Image{}
 	for info, img := range m {
 		annotations := map[string]string{
 			ocispec.AnnotationBaseImageName: info.Reference,
 		}
+		// NOTE(mkcp): It's ok if the metadata is empty / err, they're just for debug logs.
+		wStart := time.Now()
+		size, _ := img.Size() //nolint:errcheck
+		l.Info("saving image", "ref", info.Reference, "size", size, "method", "sequential")
 		if err := cl.AppendImage(img, clayout.WithAnnotations(annotations)); err != nil {
 			if err := CleanupInProgressLayers(ctx, img); err != nil {
 				message.WarnErr(err, "failed to clean up in-progress layers, please run `zarf tools clear-cache`")
+				l.Error("failed to clean up in-progress layers. please run `zarf tools clear-cache`")
```

Comment:
In Zarf now, `message.WarnErr` prints in the yellow warn format. I think it makes sense to keep in that format and just use `l.Error` for returned errors that stop the CLI entirely.

Reply:
Typically warns are used in systems to inform users but it's been handled automatically, or it don't require immediate action, e.g. we had to fallback to a slower method or a feature is deprecated. Failing to clean up here notes an actual Error and we should log it as that to users imo.

Reply:
that makes a lot of sense, thanks for the explanation!